contracts/solana/tests/trade.test.ts

```
@@ -0,0 +1,444 @@
+    try {
+      await program.methods
+        .disputeTrade()
+        .accounts({
+          trade: tradePDA,
+          disputer: unauthorizedUser.publicKey,
+        })
+        .signers([unauthorizedUser])
+        .rpc();
+      expect.fail("Expected error");
+    } catch (err: any) {
+      if (err.error && err.error.errorCode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)